.github/DISCUSSION_TEMPLATE/issue-triage.yml

```
@@ -14,22 +14,37 @@ body:
       description: |
         Provide a detailed description of the issue.  Include relevant information, such as:
         - The feature or configuration option you encounter the issue with.
-        - The expected behavior.
-        - The actual behavior (and how it deviates from the expected behavior, if it is not immediately obvious).
-        - Relevant Ghostty logs or other stacktraces.
-        - Relevant screenshots, screen recordings, or other supporting media (as needed).
+        - Screenshots, screen recordings, or other supporting media (as needed).
         - If this is a regression of an existing issue that was closed or resolved, please include the previous item reference (Discussion, Issue, PR, commit) in your description.
 
         >[!TIP]
```

You have a space everywhere else.
```suggestion
        > [!TIP]
```